packages/sit-onyx/src/components/OnyxInput/OnyxInput.ct.tsx

```diff
@@ -1,7 +1,88 @@
-import { executeScreenshotsForAllStates } from "../../utils/playwright";
+import { DENSITIES } from "../../composables/density";
 import { expect, test } from "../../playwright-axe";
+import { executeMatrixScreenshotTest } from "../../playwright/screenshots";
 import OnyxInput from "./OnyxInput.vue";
 
+test.describe("Screenshot tests", () => {
+  for (const state of ["default", "placeholder", "with-value", "autofill"] as const) {
```

Comment:
might be prettier on the screenies?
```suggestion
  for (const state of ["default", "placeholder", "with value", "autofill"] as const) {
```
(remember to adjust line 18 accordingly)